tidy3d/components/data/data_array.py

```diff
@@ -71,16 +71,23 @@ class DataArray(xr.DataArray):
     _data_attrs: Dict[str, str] = {}
 
     def __init__(self, data, *args, **kwargs):
-        """Initialize ``DataArray``."""
-
-        # initialize with untraced data
-        super().__init__(getval(data), *args, **kwargs)
-        # and put tracers in .attrs
+        super().__init__(data, *args, **kwargs)
```

Comment:
note: this inserts the dtype=object into the `values`, as we discussed. we should revisit this choice later.